lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -183,9 +187,9 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
         centerX + threshold * radius * xAngle,
         centerY + threshold * radius * yAngle,
       );
+      _titlesPosition[index] = featureOffset;
       canvasWrapper.translate(featureOffset.dx, featureOffset.dy);
-      canvasWrapper.rotate(angle * index);
-
+      canvasWrapper.rotate(data.titleAngle ?? (angle * index));
```

Comment:
I think we can make a typedef (like what you did for getTickTitle) and make data.titleAngle customizable per each index (instead of one titleAngle value)

Reply:
Will do